assets/js/components/setup/SetupUsingProxyWithSignIn.js

```diff
@@ -361,6 +361,7 @@ export default function SetupUsingProxyWithSignIn() {
 																			onClick={
```

Comment:
Like the `ResetButton`, we should also not render this until the checks are completed, otherwise we end up in a state like this on load
<img width="560" alt="image" src="https://user-images.githubusercontent.com/1621608/167014397-8192b733-3a05-491b-a295-89186fd5f489.png">